scripts/enforceVersions.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+if [[ $(pwd) =~ .+simorgh ]]; then
+  desired_node_version=$(cat .nvmrc)
+
+  if [[ $(node -v) != "$desired_node_version"* ]]; then
+    tput setaf 1
+    echo "You're not using our version of node, please move to $desired_node_version"
+    tput sgr0
+    exit 1
+  fi
+
+  desired_npm_version=$(cat .npmrc | cut -d "=" -f2)
+
+  if [[ $(npm -v) != $desired_npm_version ]]; then
+    tput setaf 1
+    echo "You're not using our version of npm, please move to $desired_npm_version"
```

Comment:
It would be good to include helpful additions in each command. e.g. `nvm install $desired_node_version && nvm use $desired_node_version` for the node error and `npm i -g npm@$desired_npm_version` for the npm error. 
In the npm case, we'd also want to say `Please 'npm install' again and commit any changes to your package-lock.json`.